lib/index.ts

```diff
@@ -483,8 +478,11 @@ export async function buildDepGraphFromImportsAndModules(
 
   try {
     const goModAbsolutPath = path.resolve(root, path.dirname(targetFile));
-    goDepsOutput = await subProcess.execute('go list', ['-json', '-deps', './...'], { cwd: goModAbsolutPath } );
+    goDepsOutput = await runGo(['list', '-json', '-deps', './...'], {cwd: goModAbsolutPath});
   } catch (err) {
+    if (/cannot find main module, but found/.test(err)) {
```

Comment:
Go 1.16 throws an error in this scenario, where previous versions just warned and was handled by [this line](https://github.com/snyk/snyk-go-plugin/pull/93/files#diff-d5a4b3a0309f2337144861084c1014523cced561eb722cb9684c63f5e41e0567R491)